deltacat/compute/compactor_v2/utils/content_type_params.py

```diff
@@ -75,11 +76,21 @@ def _download_parquet_metadata_for_manifest_entry(
     entry_index: int,
     deltacat_storage: unimplemented_deltacat_storage,
     deltacat_storage_kwargs: Optional[Dict[Any, Any]] = {},
+    file_reader_kwargs_provider: Optional[ReadKwargsProvider] = None,
 ) -> Dict[str, Any]:
+    logger.info(
+        f"Downloading the parquet metadata for Delta with locator {delta.locator} and entry_index: {entry_index}"
+    )
+    if deltacat_storage_kwargs.get("file_reader_kwargs_provider") is not None:
```

Comment:
How about using in operator here? For example, if `deltacat_storage_kwargs={"file_reader_kwargs_provider": None}`, you would still see the keyword arg conflict runtime error.

Reply:
Updated